justfile

```diff
@@ -298,6 +298,76 @@ e2e-flamegraph:
     echo "Running cargo flamegraph"
     cargo flamegraph --bin importer-online --deterministic --features dev,perf -- --external-rpc=http://localhost:3003/rpc --chain-id=2009
 
+e2e-importer-online:
+    #!/bin/bash
+
+    just e2e-importer-online-up
+    result_code=$?
+
+    just e2e-importer-online-down
+
+    exit $result_code
+
+# E2E: Importer Online
+e2e-importer-online-up:
+    #!/bin/bash
+
+    # Build Stratus and Run With Importer binaries
+    echo "Building Stratus and Run With Importer binaries"
+    cargo build --release --bin stratus --bin run-with-importer --features dev
+
+    mkdir e2e_logs
+
+    # Wait for builds to finish
+    wait
```

Comment:
cargo build is a blocking job, I don't think this wait is needed

Reply:
Done.